sdk/agrifood/agrifood-farming-rest/review/agrifood-farming.api.md

```diff
@@ -162,41 +224,41 @@ export interface ApplicationDataListByFarmerIdQueryParam {
 export interface ApplicationDataListByFarmerIdQueryParamProperties {
     $maxPageSize?: number;
     $skipToken?: string;
-    associatedBoundaryIds?: string[];
-    ids?: string[];
+    associatedBoundaryIds?: Array<string>;
+    ids?: Array<string>;
     maxArea?: number;
     maxAvgMaterial?: number;
-    maxCreatedDateTime?: Date;
-    maxLastModifiedDateTime?: Date;
-    maxOperationEndDateTime?: Date;
-    maxOperationModifiedDateTime?: Date;
-    maxOperationStartDateTime?: Date;
+    maxCreatedDateTime?: Date | string;
+    maxLastModifiedDateTime?: Date | string;
+    maxOperationEndDateTime?: Date | string;
+    maxOperationModifiedDateTime?: Date | string;
+    maxOperationStartDateTime?: Date | string;
     maxTotalMaterial?: number;
     minArea?: number;
     minAvgMaterial?: number;
-    minCreatedDateTime?: Date;
-    minLastModifiedDateTime?: Date;
-    minOperationEndDateTime?: Date;
-    minOperationModifiedDateTime?: Date;
-    minOperationStartDateTime?: Date;
+    minCreatedDateTime?: Date | string;
+    minLastModifiedDateTime?: Date | string;
+    minOperationEndDateTime?: Date | string;
+    minOperationModifiedDateTime?: Date | string;
+    minOperationStartDateTime?: Date | string;
     minTotalMaterial?: number;
-    names?: string[];
-    operationBoundaryIds?: string[];
-    propertyFilters?: string[];
-    sources?: string[];
-    statuses?: string[];
+    names?: Array<string>;
+    operationBoundaryIds?: Array<string>;
+    propertyFilters?: Array<string>;
+    sources?: Array<string>;
```

Comment:
I have to wonder why all the `X[]` types changed to `Array<X>`. Most style guides I've read prefer the former.

Reply:
Thanks Will, we have always used `Array<X>` for RLCs. Is there any difference for Typescript to use `Array<X>` vs `X[]`?

I have filed an issue to track RLC design questions and added this one. https://github.com/Azure/autorest.typescript/issues/1286

Reply:
No, there is no type-level difference, just a stylistic one.